docs/rst/tutorials/core/deployment/ds_locators_runtime/ds_locators_runtime.rst

```diff
@@ -0,0 +1,369 @@
+            Server ID:          1
+            Server GUID prefix: 44.53.01.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+The output displays the ``server id`` set, followed by the server GUID prefix.
+Server addresses ``0.0.0.0`` net mask allows all addresses to be set, but we need each server specific address.
+Finally, the ``11811`` default port would be necessary for further configuration.
+
+In order to set-up the *clients* to discover each other through the *server*, we need to set the *server* data (IP
+address, port and prefix) in all *clients*.
+The port and prefix are already known, and each specific *server* IP address can be obtained introducing the container
+names in the following command:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_0
+
+    .. tab:: Discovery Environment 1
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_1
+
+.. note::
+
+    For this example, these are the used IP addresses for each entity.
+
+    +----------------+------------+
+    | CONTAINER NAME | IP ADDRESS |
+    +================+============+
+    | client_sub     | 172.17.0.2 |
+    +----------------+------------+
+    | client_pub_0   | 172.17.0.3 |
+    +----------------+------------+
+    | client_pub_1   | 172.17.0.4 |
+    +----------------+------------+
+    | server_0       | 172.17.0.5 |
+    +----------------+------------+
+    | server_1       | 172.17.0.6 |
+    +----------------+------------+
+
+Create a XML configuration file in each *client* container, and complete it with the following XML profile, including
+the *server* data obtained (IP address, port and GUID prefix).
+
+.. literalinclude:: /resources/tutorials/core/deployment/ds_locators_runtime/client_configuration.xml
+    :language: xml
+
+The entities need to be configured to use the created profile.
+To do so, make sure the document name is ``DEFAULT_FASTRTPS_PROFILES.xml``, and set the following environment variable in all *clients*:
+
+.. code-block:: bash
+
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+``FASTDDS_ENVIRONMENT_FILE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Finally, each *client* needs to start a discovery client.
+To do so, and this is the key of the tutorial, the environment variable ``FASTDDS_ENVIRONMENT_FILE`` must be set.
+
+Setting this environment variable to an existing ``json`` file allows to load the environment variables from a file
+instead of from the environment.
+This allows to change the value of some environment variables at run time with just modifying and saving the changes
+to the file.
+
+Create a json file for each *client* and introduce the *server* IP address and port.
+Make sure the talker in the discovery environment 1 only contains its discovery server data:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
```

Comment:
I'm not sure this are the correct names for the tabs here. This JSON would let a client connect to both servers, is that what you wanted to do?

Reply:
Yes. After testing, the only way (that I founded that) it works was when the `listener` is already connected to both servers, and updating the `talker` makes the `listener` receive its publications. Here it is a truth table built based on the experiments done:

s - Server 0       T - Talker in discovery server network 1
S - Server 1       L - Listener in discovery server network 0

0 - NOT Connected to the server
1 - Connected to the server

sS  sS
01 11 - Listener DOES NOT receive Talker publications
10 11 - Listener receives Talker publications
11 10 - Listener receives Talker publications
11 11 - Listener receives Talker publications
 T    L
 
It is a really weird behaviour, but that was why I set it like that.